src/classes/Handsontable.php

```diff
@@ -0,0 +1,850 @@
+        $this->manualColumnResize = $manualColumnResize;
+    }
+
+    public function setManualRowResize($manualRowResize) {
+        $this->manualRowResize = $manualRowResize;
+    }
+
+    public function setMinSpareRows($minSpareRows) {
+        $this->minSpareRows = $minSpareRows;
+    }
+
+    public function setDataSchema($dataSchema) {
+        $this->dataSchema = $dataSchema;
+    }
+
+    public function setData($data) {
+        $this->data = $data;
+    }
+
+    public function setContainerName($containerName) {
+        $this->containerName = $containerName;
+    }
+
+    public function setLoadLibrairy($loadLibrairy) {
+        $this->loadLibrairy = $loadLibrairy;
+    }
+
+    public function setLoadDataSource($loadDataSource) {
+        $this->loadDataSource = $loadDataSource;
+    }
+
+    public function setSaveDataSource($saveDataSource) {
+        $this->saveDataSource = $saveDataSource;
+    }
+
+    public function setSave($save) {
+        $this->save = $save;
+    }
+
+    public function setAutosave($autosave) {
+        $this->autosave = $autosave;
+    }
+
+    public function setLoadAction($loadAction) {
+        $this->loadAction = $loadAction;
+    }
+
+    public function setSaveElementId($saveElementId) {
+        $this->saveElementId = $saveElementId;
+    }
+
+    public function setStartRows($startRows) {
+        $this->startRows = $startRows;
+    }
+
+    public function setStartCols($startCols) {
+        $this->startCols = $startCols;
+    }
+
+    public function setWidth($width) {
+        $this->width = $width;
+    }
+
+    public function setHeight($height) {
+        $this->height = $height;
+    }
+
+    public function setRowHeaders($rowHeaders) {
+        $this->rowHeaders = $rowHeaders;
+    }
+
+    /**
+     * Possibly values :
+     * Implemented
+     * Setting true or false will enable or disable the default column headers (A, B, C).
+     * You can also define an array ['One', 'Two', 'Three', ...]
+     * Not Implemented
+     * or a function to define the headers.
+     * @link https://docs.handsontable.com/0.35.1/Options.html#colHeaders
+     *
+     * @return mixed colHeaders value
+     */
+    public function setColHeaders($colHeaders) {
+        if (!is_array($this->colHeaders) && !is_bool($this->colHeaders)) {
+            return true; // default value
+        }
+        $this->colHeaders = $colHeaders;
+    }
+
+    public function setContextMenu($contextMenu) {
+        $this->contextMenu = $contextMenu;
+    }
+
+    public function setAutoWrapRow($autoWrapRow) {
+        $this->autoWrapRow = $autoWrapRow;
+    }
+
+    public function setMaxCols($maxCols) {
+        $this->maxCols = $maxCols;
+    }
+
+    public function getColumns() {
+        return $this->columns;
+    }
+
+    public function getCell() {
+        return $this->cell;
+    }
+
+    public function getCells() {
+        return $this->cells;
+    }
+
+    public function setCells($cells) {
```

Comment:
In this setter, you change the cells attribute by creating a new CellsConfig. So, what is the expected value of the param $cells ? 
There are no documentation, so I think that the method has a normal behavior and that I have to give a CellsConfig. But you are using the CellsConfig constructor and in the CellsConfig constructor, the expected type of $cell is mixed. In my opinion, you should documentate the setCells, maybe with an example too.